src/react-navigation/views/ScreenFallback.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Platform, StyleProp, View, ViewProps, ViewStyle } from 'react-native';
+
+type Props = {
+  visible: boolean;
+  children?: React.ReactNode;
+  enabled: boolean;
+  freezeOnBlur?: boolean;
+  style?: StyleProp<ViewStyle>;
+  collapsable?: boolean;
+};
+
+let Screens: typeof import('react-native-screens') | undefined;
+
+try {
+  Screens = require('react-native-screens');
+} catch (e) {
+  // Ignore
+}
+
+export const MaybeScreenContainer = ({
+  enabled,
+  children,
+  ...rest
+}: ViewProps & {
+  enabled: boolean;
+  hasTwoStates: boolean;
+  children?: React.ReactNode;
+}) => {
+  if (Platform.OS === 'android' && Screens?.screensEnabled()) {
+    return (
+      <Screens.ScreenContainer enabled={enabled} {...rest}>
+        {children}
+      </Screens.ScreenContainer>
+    );
+  }
```

Comment:
Looks like we don't have the navigation container on iOS (JS Tabs have it) but it works without it? 

That's weird 😅

If the `ScreenContainer` is required then adding this might not be possible. As we **need** to have multiple children here 

@tboba Could you help here? 🙏

Reply:
@okwasniewski Yeah on iOS it works even without Screen Container.

In case of Android we need `ScreenContainer` and I just checked that It worked even with `hasTwoStates={false}` on both platforms with New and Old Arch.

Reply:
Hey, @shubhamguptadream11 after consulting with `react-native-screens` team (thanks @kkafar, @tboba)  it looks like `detachInactiveScreens` won't work without the `ScreenContainer`. 

Can you change the PR to only add `freezeOnBlur`?